src/adapter/src/metrics.rs

```diff
@@ -54,7 +56,7 @@ impl Metrics {
             queue_busy_seconds: registry.register(metric!(
                 name: "mz_coord_queue_busy_seconds",
                 help: "The number of seconds the coord queue was processing before it was empty. This is a sampled metric and does not measure the full coord queue wait/idle times.",
-                buckets: histogram_seconds_buckets(0.000_128, 8.0)
+                buckets: HISTOGRAM_SECOND_BUCKETS.into(),
```

Comment:
I'd prefer this to remain a function, it just gets a higher second argument. That'll make it so changes to the const don't also change this bucketing.

Reply:
Great call, reverted it to a function!